Makefile

```diff
@@ -28,3 +28,10 @@ lint: .venv
 clean:
 	@rm -rf .venv
 	@find -iname "*.pyc" -delete
+
+.PHONY: e2e
+e2e: .venv
+	# NOTE: only the evaluation feature is run for now
+	cp test-harness/features/evaluation.feature tests/features/
+	behave tests/features/
+	rm tests/features/*.feature
```

Comment:
You may want to add a target that's a prereq to this one that pulls the submodule.

I know the CI does this, and you should keep that, but it's "nice" to have to have it done automatically to make it easy for devs running it locally (people tend to have trouble with git sub-modules in my experience :sweat_smile: ).